### PR TITLE
fix: bump supported versions

### PR DESCRIPTION
**Pull request by APshenkin**
Fixes https://github.com/biomejs/biome-intellij/issues/74

### CODE REVIEW

**Comment by APshenkin** [comment]
@ematipico I'll appreciate if you can review and publish this 🙏

File: gradle.properties [hunk 1]
```diff
@@ -3,13 +3,13 @@ pluginGroup=com.github.biomejs.intellijbiome
 pluginName=Biome
 pluginRepositoryUrl=https://github.com/biomejs/biome
 # SemVer format -> https://semver.org
-pluginVersion=1.2.1
+pluginVersion=1.2.2
```
Comment (ematipico) [inline gradle.properties:6]: Can you please revert this? This will be bumped during the release

**Review by ematipico — CHANGES_REQUESTED** [review]
Build is failing, do you know why?

**Comment by APshenkin** [comment]
@ematipico Hm this looks weird. Unfortunately I can't build it on my machine either, as have some corporate tools, blocking me to fetch some deps..., so I hoped that just bumping version will do the thing without testing it locally :( 

I'll appreciate, if you will have an option to try to build it locally

**Comment by ematipico** [comment]
I believe that the latest version of IntelliJ changed few things, and it requires some code change. In fact, there are other errors in CI. 

Unfortunately, I don't have the knowledge and time to do so.

**Pull request closed without merging**